x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/search_index/connector/connector_scheduling.tsx

```diff
@@ -200,25 +203,51 @@ export const ConnectorSchedulingComponent: React.FC = () => {
         </EuiFlexItem>
         {shouldShowAccessControlSync && (
           <EuiFlexItem>
-            <SchedulePanel
-              title={i18n.translate(
-                'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.title',
-                { defaultMessage: 'Document Level Security' }
-              )}
-              description={i18n.translate(
-                'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.description',
-                {
-                  defaultMessage:
-                    'Control the documents users can access, based on their permissions and roles. Schedule syncs to keep these access controls up to date.',
-                }
+            <EuiFlexGroup direction="column">
+              <EuiFlexItem>
+                <SchedulePanel
+                  title={i18n.translate(
+                    'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.title',
+                    { defaultMessage: 'Document Level Security' }
+                  )}
+                  description={i18n.translate(
+                    'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.description',
+                    {
+                      defaultMessage:
+                        'Control the documents users can access, based on their permissions and roles. Schedule syncs to keep these access controls up to date.',
+                    }
+                  )}
+                >
+                  <ConnectorContentScheduling
+                    type={SyncJobType.ACCESS_CONTROL}
+                    index={index}
+                    hasPlatinumLicense={hasPlatinumLicense}
+                  />
+                </SchedulePanel>
+              </EuiFlexItem>
+              {isDocumentLevelSecurityDisabled && (
+                <EuiFlexItem>
+                  <EuiCallOut
+                    title={i18n.translate(
+                      'xpack.enterpriseSearch.content.indices.connectorScheduling.schedulePanel.documentLevelSecurity.dlsDisabledCallout.title',
+                      { defaultMessage: 'Permissions and identity syncs not allowed' }
```

Comment:
```suggestion
                      { defaultMessage: 'Access control syncs not allowed' }
```
Is this right name? :)